.depcheckrc.yml

```diff
@@ -1,6 +1,7 @@
 ---
 ignores:
   # monorepo
+  - '@metamask/kernel-shims'
```

Comment:
Depcheck started complaining after removing the extension's unused `endoify.ts`.